docs/src/extend/plugins.md

```diff
@@ -89,6 +89,10 @@ Plugin environments can define the following objects:
 
 ### Processors in Plugins
 
+::: warning
+File Extension-named Processors are deprecated and only work in eslintrc-style configuration files. Flat config files do not automatically apply processors; you need to explicitly set the `processor` property.
```

Comment:
Fixing capitalization.

```suggestion
File extension-named processors are deprecated and only work in eslintrc-style configuration files. Flat config files do not automatically apply processors; you need to explicitly set the `processor` property.
```

Reply:
I was matching the capitalization in the section title at [`custom-processors.md#140`](https://github.com/eslint/eslint/blob/53349cae6eb43d0ecb9c267633bf5bcc03660ac1/docs/src/extend/custom-processors.md?plain=1#L140) for consistency, as it seemed like the "proper" name of the feature. Happy to go either way, just pointing out that it was intentional this way 😄